JSDemos/Demos/TabPanel/Overview/jQuery/index.html

```diff
@@ -13,31 +13,33 @@
     <script src="index.js"></script>
   </head>
   <body class="dx-viewport">
-    <div class="tabpanel-demo">
-      <div class="widget-container">
-        <div id="tabpanel" class="dx-theme-background-color"></div>
-      </div>
+    <div class="demo-container">
+      <div class="tabpanel-demo">
```

Comment:
I suggest we may delete 'tabpanel-demo' wrapper and use default .demo-container wrapper. It's common wrapper container that used in all approaches.

Reply:
If I add display: flex on the demo-container, my layout breaks (in framworks), since the main point of tabpanel-demo is to be a flex-wrapper distributing content (on the left side) and options (on the right side).

```
<body class="dx-viewport">
     <div class="demo-container">
           <div id="app"></div>
      </div>
</body>
```

Adding flex styles to the demo-container will not work, because inside the .demo-container is <div id="app"></div>

In jQuery we can choose not to add tabpanel-demo element, but then the layout will be different from other frameworks. I don't think that's the right way to do it.